lib/transaction/pool.go

```diff
@@ -2,10 +2,15 @@ package transaction
 
 import (
 	"sync"
+	"time"
 
+	"github.com/ChainSafe/gossamer/dot/metrics"
 	"github.com/ChainSafe/gossamer/lib/common"
 )
 
+const collectTxMetricsTimeout = time.Second * 5
```

Comment:
It would be nice to have this defined in the `NewPool` constructor rather than package level const.  You could even create a private `newPool` that takes in configurable durations for this so you can adjust them in the tests.

Reply:
I've removed this and used the metrics default value